<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutToolWidget</class>
 <widget class="QDialog" name="LayoutToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layout Tool</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LayoutMain" stretch="6,4,5,4">
    <item>
     <widget class="QGroupBox" name="GBAlignment">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Alignment</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="LayoutAlign">
        <item row="0" column="0">
         <widget class="QPushButton" name="BtnLeft">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="BtnCenter">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Center</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="BtnMiddle">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Middle</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="BtnTop">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="BtnRight">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="BtnBottom">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="GBSameLength">
      <property name="title">
       <string>Make Same Length</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>20</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="LayoutSameLength">
        <item>
         <widget class="QPushButton" name="BtnWidth">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>WIdth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnHeight">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnBoth">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Both</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="GBDistribute">
      <property name="title">
       <string>Distribute</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>68</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="LayoutDistribute">
        <item>
         <widget class="QPushButton" name="BtnHorizon">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Horizontally with Equal Spacing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnVertical">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Vertically with Equal Spacing</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="GBHelper">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Helper</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="LayoutHelper">
        <item>
         <widget class="QPushButton" name="BtnAraange">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Arrange...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnReorder">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Re-order</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
